restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/changelog_depreciated/mariadb/update_script_Aug07_2017.sql

```diff
@@ -0,0 +1,48 @@
+#SQL Script for MySQL/MariaDB to update DB with the schema changes
```

Comment:
@agafox you are right this new feature should be in the public repository. The DB schema upgrade script for existing customer deployments should be in the private repository